src/20230920_AaveV3_Eth_AaveImmunefiActivation/AaveV3_Ethereum_AaveImmunefiActivation_20230920.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+
+/**
+ * @title Aave <> Immunefi activation
+ * @author BGD Labs @bgdlabs
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0xb477feee6d506be940154bc84278654f2863044854f80fca825b236253a97778
+ * - Discussion: https://governance.aave.com/t/bgd-aave-immunefi-bug-bounty-program/14757
+ */
+contract AaveV3_Ethereum_AaveImmunefiActivation_20230920 is IProposalGenericExecutor {
+  function execute() external {
```

Comment:
Just an idea but do you want to emit an event detailing the agreement to have a log on-chain? Not really necessary, but could be a neat pattern

Reply:
Done on https://github.com/bgd-labs/aave-proposals/pull/308/commits/62edce10d3bf20f37b743de6a70c12814d259b44